src/writing-policies/rego/gatekeeper/03-build-and-run.md

```diff
@@ -0,0 +1,82 @@
+# Build and run
+
+Building and running the policy is done exactly the same as a Rego
+policy targeting Open Policy Agent. The structure is like:
+
+```
+.
+├── data
+│   ├── default-ns.json
+│   └── other-ns.json
+└── policy.rego
+
+1 directory, 3 files
+```
+
+## Build
+
+Let's build our policy by running the following `opa` command:
+
+```shell
+~/gatekeeper-policy » opa build -t wasm -e policy/violation policy.rego
+```
+
+What this does is build the rego policy, with:
+
+- `target`: `wasm`. We want  to build the policy for the `wasm` target.
+- `entrypoint`: `policy/main`. The entry point is the `main` rule
+inside the `policy` package.
+- `policy.rego`: build and include the `policy.rego` file.
+- `request.rego`: build and include the `request.rego` file.
+
+After the build is complete, `opa build` will have generated a
+`bundle.tar.gz` file. You can extract it:
+
+```shell
+gatekeeper-policy » tar -xf bundle.tar.gz /policy.wasm
+```
+
+The tree looks like:
+
+```
+.
+├── bundle.tar.gz
+├── data
+│   ├── default-ns.json
+│   └── other-ns.json
+├── policy.rego
+└── policy.wasm
+
+1 directory, 5 files
+```
+
+We can now execute our policy!
+
+## Run
+
+Let's use `kwctl` to run our policy as follows:
+
+```
+gatekeeper-policy »  kwctl run -e gatekeeper --request-path data/other-ns.json policy.wasm | jq
+{
+  "uid": "1299d386-525b-4032-98ae-1949f69f9cfc",
+  "allowed": true
+}
+```
+
+Given that this is our resource created in the namespace called
+`other`, this resource is accepted, as expected. Now let's execute a
+request that will be rejected by the policy:
+
+```
+gatekeeper-policy »  kwctl run -e gatekeeper --request-path data/default-ns.json policy.wasm | jq
+{
+  "uid": "1299d386-525b-4032-98ae-1949f69f9cfc",
+  "allowed": false,
+  "status": {
+    "message": "it is forbidden to use the default namespace"
+  }
+}
+```
+
+As you can see, our Gatekeeper policy rejected this resource as expected.
```

Comment:
I would add a section talking about builtins to express the following concepts:

* Some Rego functions must be provided by the runtime, hence us. See the builtins section we already have
* Important: kwctl run will exit with an error if one of the builtins required by the policy is not supported. This is a reliable information (meaning, we know about that for sure, it doesn't depend on the execution path the policy has)